data/schema/units/abilities.cfg

```diff
@@ -5,6 +5,8 @@
 		max=infinite
 		super="units/unit_type/abilities/~generic~,units/unit_type/attack/specials/" + {NAME}
 		{FILTER_TAG "filter_student" unit {FILTER_TAG "filter_weapon" weapon ()}}
+		{FILTER_TAG "filter_student_adjacent" adjacent ()}
+		{FILTER_TAG "filter_student_adjacent_location" adjacent_location ()}
```

Comment:
For consistency with [filter_adjacent_location], I think these should be renamed to [filter_adjacent_student] and [filter_adjacent_student_location].

Reply:
done